app/models/manageiq/providers/autosde/storage_manager/host_initiator.rb

```diff
@@ -0,0 +1,21 @@
+class ManageIQ::Providers::Autosde::StorageManager::HostInitiator < ::HostInitiator
+  supports :create
+
+  def self.raw_create_host_initiator(_ext_management_system, _options = {})
+    host_initiator_to_create = _ext_management_system.autosde_client.StorageHostCreate(
+      :name           => _options['name'],
+      :port_type      => _options['port_type'],
+      :storage_system => PhysicalStorage.find(_options['physical_storage_id']).ems_ref,
+      :iqn            => _options['iqn'] || "",
+      :wwpn           => _options['wwpn'] || "",
+      :chap_name      => _options['chap_name'] || "",
+      :chap_secret    => _options['chap_secret'] || ""
```

Comment:
I assume we're going to need DDF forms for these params?  cc @skateman

Reply:
I don't really get the context, but probably...

Reply:
@agrare @skateman I did the DDF forms, you can look on it in https://github.com/ManageIQ/manageiq-ui-classic/pull/7587/files#diff-9b1b0595c256080f1cd7cbc9d1d01f2d32608b561cf00f0667b4a76afc8f8e35R1-R115